internal/pubsub/pubsub.go

```diff
@@ -202,19 +194,16 @@ func (s *Subscription) Close() error {
 // It is for use by provider implementations.
 func NewSubscription(ctx context.Context, d driver.Subscription) *Subscription {
 	handler := func(item interface{}) {
-		mecs := item.([]msgErrChan)
+		ms := item.([]*Message)
 		var ids []driver.AckID
-		for _, mec := range mecs {
-			m := mec.msg
+		for _, m := range ms {
 			id := m.ackID
 			ids = append(ids, id)
 		}
-		err := d.SendAcks(ctx, ids)
```

Comment:
It looks like this is no longer calling `SendAcks` at all...?

Reply:
Huh? There is a call to SendAcks still.

Reply:
Ah sorry, it is down below, I missed it.